rust/lance-index/src/vector/ivf/shuffler.rs

```diff
@@ -470,6 +603,8 @@ impl IvfShuffler {
                     "Chunk loaded into memory and sorted, writing to disk at {}",
                     path
                 );
+
+                // TODO: The result can be lance v1 or v2. Currently it is v1
```

Comment:
Currently the sorted files are written using v1, this should be updated eventually